Yoast/wordpress-seo
# add --no-git-push option to sync-gutenberg-versions grunt task

## Context
* add --no-git-push option to sync-gutenberg-versions grunt task needed for RC beta release "pre" run process.
* applied code style to the changed file.


## Summary

This PR can be summarized in the following changelog entry:

* minor change:  add test option to a automation task

## Relevant technical choices:

* default to old behavior

## Test instructions

**Do not TEST** without the new `--no-git-push` on yoast\wordpress-seo
this will push your current changes to the current release branch
this is also kind of why this PR is made so you can test without doing a live push.
aka if You want to run without the new parameter you first need to **fork** this repo and do tests there.

**test _with_ option set:**
* set the version of gutenberg in the `admin/class-gutenberg-compatibility.php` file to an older version of gutenberg
* run this command: `grunt sync-gutenberg-version --no-git-push`
* the version in `admin/class-gutenberg-compatibility.php` should now be changed back the to the current version, commit ed to your current branch, but not pushed

**test _without_ option set:**
* please make sure you have **forked**, see the the note from above
* also make sure you have checked out this branch on your fork
* set the version of gutenberg in the `admin/class-gutenberg-compatibility.php` file to an older version of gutenberg
* run this command: `grunt sync-gutenberg-version`
* the version in `admin/class-gutenberg-compatibility.php` should now be changed back the to the current version, commit ed to your current branch and pushed to github (on your fork)

### Test instructions for the acceptance test before the PR gets merged
This PR can be acceptance tested by following these steps:

*


### Test instructions for QA when the code is in the RC




QA can test this PR by following these steps:

* See that the current RC contains the latest version of the Gutenberg plugin in `wordpress-seo/admin/class-gutenberg-compatibility.php` (lines 18 and 25).
    * Or more specifically, contains the version that was the latest version on the day that the RC was created. It is possible that a new release of the Gutenberg plugin has taken place in the mean time.

## Impact check

This PR affects the following parts of the plugin, which may require extra testing:

*

## UI changes


## Documentation


## Quality assurance


Fixes #
